src/SpeciesCard.js

```diff
@@ -1,13 +1,41 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import EbiSpeciesIcon from 'react-ebi-species'
 
-const SpeciesCard = () => {
+const SpeciesCard = ({iconSrc, description, content}) => {
   return (
-    <p>SpeciesCard content</p>
+    <div className={`column column-block text-center combo card`} style={{marginBottom:0, paddingBottom: `25px`}}>
+      <span className={`species-icon`} style={{fontSize: `600%`}}>
+        <EbiSpeciesIcon species={iconSrc}/>
+      </span>
+
+      {
+        description && <h5 className="species-name">{description}</h5>
```

Comment:
Stick to either `className="foo"` or `className={foo-wrapped-in-backquotes}`. I can see both around here.

Reply:
Yeah, I think I missed a few out. I went through most of them and tried to make them consistent. Not sure I like the `{`class-name`} approach, but having double quotes breaks the backtick rule. Decisions decisions